### ISSUE TITLE
Report portal stopped working after running robot framework listener more than 10 hours

### ISSUE THREAD

**Issue by lizzyliuY** [reportportal/agent-Python-RobotFramework#132]
---
name: Bug report
about: Create a report to help us improve
title: ''
labels: bug
assignees: ''

---

**Describe the bug**
We have about 15000 cases running on 30 controllers. I installed listener on controllers and adding variables to pybot command to run those cases. There have about 4-5 tasks on each controller, total 120-150 tasks. After running about 10 hours, postgres image and rabbitmq image stopped working.
root@controller:~/reportportal/data/postgres/base# docker ps
CONTAINER ID   IMAGE                                                  COMMAND                  CREATED        STATUS                          PORTS                                                                   NAMES
49bea56efc52   reportportal/service-api:5.5.0                         "/bin/sh -c ./start.…"   11 hours ago   Up 11 hours                     8080/tcp                                                                reportportal_api_1
47876f47a502   reportportal/service-index:5.0.10                      "/service-index"         11 hours ago   Up 11 hours                     8080/tcp                                                                reportportal_index_1
ecf2850bbfd9   reportportal/service-metrics-gatherer:1.1.14           "/venv/bin/uwsgi --h…"   11 hours ago   Up 11 hours (unhealthy)         5000/tcp                                                                reportportal_metrics-gatherer_1
e4f318c50c1f   reportportal/service-auto-analyzer:5.5.0               "/venv/bin/uwsgi --h…"   11 hours ago   Up 11 hours (unhealthy)         5001/tcp                                                                reportportal_analyzer_train_1
45b71b129b2a   reportportal/service-auto-analyzer:5.5.0               "/venv/bin/uwsgi --h…"   11 hours ago   Up 11 hours (unhealthy)         5001/tcp                                                                reportportal_analyzer_1
2e6a276dbb77   minio/minio:RELEASE.2020-10-27T04-03-55Z               "/usr/bin/docker-ent…"   11 hours ago   Up 11 hours (healthy)           9000/tcp                                                                reportportal_minio_1
9d8a7460f6d0   docker.elastic.co/elasticsearch/elasticsearch:7.10.1   "/tini -- /usr/local…"   11 hours ago   Up 11 hours (healthy)           9200/tcp, 9300/tcp                                                      reportportal_elasticsearch_1
60dd560aa6b8   reportportal/service-jobs:5.5.0                        "/bin/sh -c ./start.…"   11 hours ago   Up 11 hours                     8080/tcp                                                                reportportal_jobs_1
e01dd562b8db   reportportal/service-authorization:5.5.0               "/bin/sh -c ./start.…"   11 hours ago   Up 11 hours                     8080/tcp                                                                reportportal_uat_1
451f8b6a714e   postgres:12-alpine                                     "docker-entrypoint.s…"   11 hours ago   Restarting (1) 8 seconds ago                                                                            reportportal_postgres_1
fabee20e97aa   traefik:v2.0.7                                         "/entrypoint.sh --pr…"   11 hours ago   Up 11 hours                     80/tcp, 0.0.0.0:8080-8081->8080-8081/tcp, :::8080-8081->8080-8081/tcp   reportportal_gateway_1
af2ec32955c0   rabbitmq:3.7.16-management                             "docker-entrypoint.s…"   11 hours ago   Restarting (4) 25 seconds ago                                                                           reportportal_rabbitmq_1
d770ef5ad1f8   reportportal/service-ui:5.5.0                          "/service-ui"            11 hours ago   Up 11 hours                     8080/tcp                                                                reportportal_ui_1

Is there a way only to log robotframework results other than log debug logs? Something like change log level?

**Comment by iivanou** [comment]
@lizzyliuY , Yes. You can use logger.info instead of logger.debug for example.

**Comment by lizzyliuY** [comment]
> @lizzyliuY , Yes. You can use logger.info instead of logger.debug for example.

Thanks for the reply, Ivan. Where should I set the log level? I ran the listener with my pybot command, in that command I need to set the log to trace. Any place to set in report portal part about the log level? If I include all the logs the total data files for postgres db is about 20G probably it is the root cause of the issue I reported.

**Comment by iivanou** [comment]
The agent utilizes RF logging functionality. You can simply change the log level via [RF instruments](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#setting-log-level).

**Comment by HardNorth** [comment]
Answered
